frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/review-tab/storage-review.tsx

```diff
@@ -50,6 +50,16 @@ const StorageReviewFirehose: React.FC<StorageReviewFirehoseProps> = ({
   persistentVolumeClaims,
   storageClasses,
 }) => {
+  const headers = [
+    { title: 'Name' },
+    { title: 'Interface' },
+    { title: 'Size' },
+    { title: 'Storage Class' },
+    { title: 'Access Mode' },
```

Comment:
can we change this from access mode to just access, same for volume mode? It helps with the table layout when on smaller viewports

Reply:
I can, but the term 'volume' has specific meanings in storage. Do you think that will be a problem?

Reply:
hmmm I'm not sure. It has context with the values so maybe not? I guess we'd want to ask someone like @aglitke who would have more insight on this.

Reply:
adding a screenshot for reference. You can see with the longer titles we get an overlay.
<img width="676" alt="Screen Shot 2020-04-17 at 10 15 15 AM" src="https://user-images.githubusercontent.com/287880/79573025-5652bf00-8094-11ea-9c0d-c32842b64de0.png">

Reply:
I think you need to keep the "mode" there. Shortening volume mode to volume feels a bit like shortening "persistent volume claim" to "persistent volume" - it changed the meaning to something else.

Reply:
I think "Access" could be okay but can't think of a way to shorten "Volume Mode".  I also see that "Filesystem" is wrapping.  Would it be enough to shorten just the Access Mode column?

Reply:
I can add the `wrappable` transform to make the headers wrap. Would that be sufficient, @matthewcarleton?

Reply:
ya we can follow up on this if it's too much effort to get in but I'd like to see the table cleaned up a bit.

Reply:
+1 to what @aglitke is saying about the access mode. That will help with the spacing. I wonder too if we can drop access from "shared access (RWX)"

Reply:
Ok. Dropping "Mode" from the "Access" header. The other options that may show up in that column are `Single User (RWO)` and `Read Only (ROX)`. I don't think those can be shortened and retain their meaning.